tests/unit/s2n_tls13_server_cert_test.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include <string.h>
+#include <stdio.h>
+#include <s2n.h>
+
+#include "stuffer/s2n_stuffer.h"
+#include "testlib/s2n_testlib.h"
+#include "tls/s2n_tls.h"
+#include "utils/s2n_safety.h"
+
+#define S2N_BLOB_FROM_HEX( name, hex )                  \
+    struct s2n_stuffer name##_stuffer;                  \
+    s2n_stuffer_alloc_ro_from_hex_string(               \
+        &name##_stuffer, hex);                          \
+    struct s2n_blob name = name##_stuffer.blob;
+
+#define S2N_BLOB_FREE( name ) do {                       \
+    EXPECT_SUCCESS(s2n_stuffer_free(&name##_stuffer));   \
+} while (0)
+
```

Comment:
Since these macros are being used in more than one unit test maybe we can move it to https://github.com/awslabs/s2n/blob/master/tests/testlib/s2n_testlib.h instead of duplicating it?

Reply:
yes, I plan to refactor these function in a later PR, also to remove the stuffer dependency creating hex blobs.

Reply:
I've added a commit to move the macros to testlib